i18n/generate.py

```diff
@@ -89,11 +93,38 @@ def clean_pofile(path):
     pomsgs = pofile(path)
     # The msgcat tool marks the metadata as fuzzy, but it's ok as it is.
     pomsgs.metadata_is_fuzzy = False
+
     for entry in pomsgs:
         # Remove line numbers
         entry.occurrences = [(filename, None) for filename, __ in entry.occurrences]
         # Remove -format flags
         entry.flags = [f for f in entry.flags if not f.endswith("-format")]
+        # Check for merge conflicts. Pick the first, and emit a warning.
+        if 'fuzzy' in entry.flags:
+            LOG.warn(
+                'Multiple translations found for single string.\n\tString "%s"\n\tPresent in files %s',
+                entry.msgid,
+                [f for (f, __) in entry.occurrences]
+            )
+            for msgstr in DUPLICATE_ENTRY_PATTERN.split(entry.msgstr):
+                # Ignore any empty strings that may result from the split call
+                if msgstr:
+                    # Set the first one we find to be the right one. Strip to remove extraneous
+                    # new lines that exist.
+                    entry.msgstr = msgstr.strip()
+
+                    # Raise error if there's new lines starting or ending the id string.
+                    if entry.msgid.startswith('\n') or entry.msgid.endswith('\n'):
+                        raise ValueError(
+                            u'{} starts or ends with a new line character, which is not allowed. '
+                            'Please fix before continuing. Source string is found in {}'.format(
+                                entry.msgid, entry.occurrences
+                            ).encode('utf-8')
+                        )
+
+                    entry.flags = [f for f in entry.flags if 'fuzzy' not in f]
```

Comment:
this is under a `if 'fuzzy' in entry.flags:` code block, so it's only being removed if it's found.